src/applications/login/components/SignInButtons.jsx

```diff
@@ -48,33 +57,35 @@ const LoginGovButtons = ({ isDisabled }) => (
       disabled={isDisabled}
       type="button"
       aria-label="Sign in with My HealtheVet"
-      className="usa-button usa-button-big mhv-button vads-u-margin-y--1p5"
+      className="usa-button mhv-button vads-u-margin-y--1p5 vads-u-padding-y--2"
       onClick={() => loginHandler('mhv')}
     >
       My HealtheVet
     </button>
     <div id="create-account">
       <h2 className="vads-u-margin-top--3">Or create an account</h2>
       <div className="vads-u-display--flex vads-u-flex-direction--column">
-        <button
-          type="button"
-          aria-label="Create an account with Login.gov"
+        <a
+          href={loginGovSignupUrl}
+          className="vads-c-action-link--blue logingov"
+          aria-label="Create an account with Login.gov. (Opens in a new window)"
```

Comment:
Does this open in a new window?

Reply:
Checked the review instance, doesn't appear to open in a new window

Reply:
@timwright12 I will update the language to "Create an account on the Login.gov website"